sponnet/pipeline.libsonnet

```diff
@@ -159,7 +159,14 @@
       withSource(source):: self + { source: source },
       addPayloadConstraint(key, value):: self + { payloadConstraints +: super.payloadConstraints + { [key]: value }},
     },
-
+    pubsub(name, pubsubSystem, subscriptionName):: trigger(name, 'pubsub') {
+      pubsubSystem: pubsubSystem,
+      subscriptionName: subscriptionName,
+      attributeConstraints: {},
+      addAttributeConstraints(key, value):: self + {attributeConstraints +: {[key]: value}},
```

Comment:
nit: just needs added spacing between object curly braces and object keys or values. Also on line 168.

Reply:
Good idea! Done.